apps/sasquatch/build.gradle

```diff
@@ -63,12 +63,15 @@ repositories {
 dependencies {
     implementation "com.android.support:support-v4:${rootProject.ext.supportLibVersion}"
     implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation "com.android.support:customtabs:${rootProject.ext.supportLibVersion}"
 
     projectDependencyImplementation project(':sdk:appcenter-analytics')
     projectDependencyImplementation project(':sdk:appcenter-crashes')
     projectDependencyImplementation project(':sdk:appcenter-distribute')
+    projectDependencyImplementation project(':sdk:appcenter-identity')
     projectDependencyImplementation project(':sdk:appcenter-push')
     projectDependencyImplementation project(':sdk:appcenter-rum')
+    projectDependencyImplementation project(':sdk:appcenter-identity')
```

Comment:
duplicate

Reply:
This is a merge PR. What happened here? 🤔

Reply:
```suggestion
```